packages/react-cards/src/components/Card/Card.view.tsx

```diff
@@ -81,14 +81,24 @@ export const CardView: ICardComponent['view'] = props => {
     }
   );
 
+  const _onKeyDown = (ev: React.KeyboardEvent): void => {
+    if (onClick && (ev.which === KeyCodes.enter || ev.which === KeyCodes.space)) {
+      onClick(ev as any);
```

Comment:
1) i think we should add a `onKeyDown` prop and invoke that instead `onClick`
2) we should not be casing to `any` unless we have to (have a good reason to)

Reply:
@xugao I've made some changes, please review again when you get the chance.